### DISCUSSION TITLE
Ability To Mute Audio

### DISCUSSION THREAD

**Discussion by JAng-A3** [Voltstro-Studios/UnityWebBrowser#357] [Requests]
### What platform are you experiencing this issue on?

Windows x64

### What version of UWB are you using?

2.2.1

### What Unity version are you running?

2023.2.12f

### Describe what the issue you are experiencing is.

Hello, I am just wondering if it is at all possible to mute the audio coming from the web browser? I know that currently UWB does not have a direct audio control function as of now, but if there is a way to modify the code so that it does not provide any audio (either by triggering a function or by default), that would be a great help to what I am currently trying to achieve using the plugin.

### Provide reproducible steps for this issue.

1. Set up a UWB Basic Client with a YouTube video as the initial URL
2. Play the application and allow the client to play, then play the video

**Comment by Voltstro** [comment]
2.2.3 has been released and includes the [`AudioMute`](https://projects.voltstro.dev/UnityWebBrowser/latest/api/voltstrostudios.unitywebbrowser.core.webbrowserclient/#VoltstroStudios_UnityWebBrowser_Core_WebBrowserClient_AudioMute_System_Boolean_) method.
